src/components/Ffuf/FfufTool.tsx

```diff
@@ -7,7 +7,10 @@ import { UserGuide } from "../UserGuide/UserGuide";
 import { SaveOutputToTextFile_v2 } from "../SaveOutputToFile/SaveOutputToTextFile";
 import { LoadingOverlayAndCancelButton } from "../OverlayAndCancelButton/OverlayAndCancelButton";
 
```

Comment:
Need to import RenderComponent to replace UserGuide. Refer to Airbase-NG. You also need to important checkAllCommandsAvailability and InstallationModal.